public/assets/css/icons/rss.css

```diff
@@ -0,0 +1,47 @@
+.icon .rss {
+    width: 32px;
+    height: 33px;
+    background: #ea4335;    
+}
+
+.rss .icon-part-one {
+    width: 20px;
+    height: 20px;
+    border-radius: 0 0 100%;    
+    background: white;
+    transform:translate3d(40%, 20%, 0) rotate(-90deg);
```

Comment:
the propriety of border radius take 4 value instead of 3.
exemple: 
```css 
border-radius: 0 0 100% 0;                                                                                    
```
[https://tympanus.net/codrops/css_reference/border-bottom-left-radius/](https://tympanus.net/codrops/css_reference/border-bottom-left-radius/)

Reply:
correction done